src/main/java/org/jenkinsci/plugins/durabletask/ProcessLiveness.java

```diff
@@ -94,14 +103,49 @@ private static boolean _isAlive(VirtualChannel channel, int pid, Launcher launch
             this.pid = pid;
         }
         @Override public Boolean call() throws RuntimeException {
-            // JNR-POSIX does not seem to work on FreeBSD at least, so using JNA instead.
+            // Try jnr's waitpid
```

Comment:
It makes sense to add `//TODO: Replace by the direct call once Jenkins core is updated to 2.64+ with https://github.com/jenkinsci/jenkins/pull/2904`

Reply:
Done.